setup.py

```diff
@@ -24,6 +24,7 @@
     "aioresponses~=0.6.0",
     "moto~=1.3.8",
     "fakeredis~=1.0",
+    "mongomock~=3.18.0",
```

Comment:
not `~=3.18`?

Reply:
I believe they're the same thing 
https://jubianchi.github.io/semver-check/#/~%3D3.18.0/3.18.23

Reply:
I believe they're the same thing
https://jubianchi.github.io/semver-check/#/~%3D3.18.0/3.18.23

Reply:
Nope, https://www.python.org/dev/peps/pep-0440/#compatible-release

Reply:
👍